cmd/inspector/main.go

```diff
@@ -244,6 +254,59 @@ func NodeInfo(cmd *cobra.Command, args []string) (err error) {
 	return nil
 }
 
+func dotOutput(nodeid string) (*os.File, error) {
+	if DotPath == "stdout" {
+		return os.Stdout, nil
+	}
+	if DotPath == "" {
+		DotPath = fmt.Sprintf("routing-graph-%s.dot", nodeid)
+	}
+
+	return os.Create(DotPath)
+}
+
+// DrawTableAsGraph outputs the table routing as a graph
+func DrawTableAsGraph(cmd *cobra.Command, args []string) (err error) {
+	i, err := NewInspector(*Addr, *IdentityPath)
+	if err != nil {
+		return ErrInspectorDial.Wrap(err)
+	}
+
+	// first lookup the node to get its address
+	n, err := i.kadclient.LookupNode(context.Background(), &pb.LookupNodeRequest{
+		Id: args[0],
+	})
+	if err != nil {
+		return ErrRequest.Wrap(err)
+	}
+
+	// now ask the node directly for its node info
+	info, err := i.kadclient.DrawTable(context.Background(), &pb.DrawTableRequest{
+		Id:      n.GetNode().Id,
+		Address: n.GetNode().GetAddress(),
+	})
+	if err != nil {
+		return ErrRequest.Wrap(err)
+	}
+	fh, err := dotOutput(args[0])
+	if err != nil {
+		panic(err)
+	}
+	defer func() {
+		err := fh.Close()
+		if err != nil {
+			panic(err)
+		}
```

Comment:
Again, instead of panicking, we usually combine `err` (from the return value declaration) with the error returned from `fh.Close()`, i.e. `defer func() { err = errs.Combine(err, fh.Close()) }()`

`errs.Combine` comes frome the `github.com/zeebo/errs` package.